www/src/pages/docs/30_Render HTML.md

```diff
@@ -672,6 +672,48 @@ render {
 }
 ```
 
+### Handle large Class Strings
+
+CSS classes strings can become very long and may exceed the limit of a code line. This is especially true when working
+with utility based CSS frameworks like [tailwindcss](https://tailwindcss.com/).
+
+For such situations you need to split up your string into feasible, shorter parts. For this use-case fritz2 offers 
+the dedicated function `joinClasses(vararg classes: String?): String`. This functions accepts an arbitrary 
+amount of `String`s and concatenates them together taking care of the needed white spaces:
+```kotlin
+// example uses tailwindcss classes
+render {
+    div(
+        joinClasses(
+            "relative z-10 flex justify-between items-start w-full my-2 p-4",
+            "bg-primary-800 rounded-lg hover:bg-primary-900",
+            "text-left text-white",
+            "focus:outline-none focus:ring-4 focus:ring-primary-600",
+        )
+    ) {
+        // some content
+    }
+}
+```
+
```

Comment:
I think a short paragraph about the handling of `null` values and the resulting best-practices regarding conditional styles may be helpful as well. E.g. `"some-class".takeIf { ... }` or `if (...) "some-style" else "another-style"`.

You may take the examples straight from the function's KDoc.